mlrun/db/httpdb.py

```diff
@@ -3408,6 +3408,7 @@ def _verify_background_task_in_terminal_state():
             logger,
             False,
             _verify_background_task_in_terminal_state,
+            fatal_exceptions=(mlrun.errors.MLRunAccessDeniedError,),
```

Comment:
I get this as a general enhancement to verify we don't retry unrecoverable situations, but what is the situation that you have in mind here? Specific case where this would get access denied due to the changes introduced?